src/components/resource-picker/wikimedia-commons-files-grid-viewer.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Spin, Tooltip } from 'antd';
+import { useTranslation } from 'react-i18next';
+import PreviewIcon from '../icons/general/preview-icon.js';
+import { wikimediaFileShape } from '../../ui/default-prop-types.js';
+import WikimediaCommonsIcon from '../icons/wikimedia-commons/wikimedia-commons-icon.js';
+import ActionButton, { ActionButtonGroup, ACTION_BUTTON_INTENT } from '../action-button.js';
+
+function WikimediaCommonsFilesGridViewer({
```

Comment:
Honestly since there is only one type of viewer for wikimedia (and since it's dedicated), we could shorten the component to `WikimediaCommonsFilesViewer`? What do you think.

Reply:
Yes, it's still long enough :laughing:, done!